LiteCore/Support/SecureDigest.hh

```diff
@@ -105,6 +105,58 @@
     }
 
     #define SECURE_DIGEST_AVAILABLE 1
+    
+#elif defined(_CRYPTO_MBEDTLS)
+    #include <mbedtls\md5.h>
```

Comment:
Fix the backslashes here, then everything else looks good so go ahead and merge!